src/Main.java

```diff
@@ -2,16 +2,172 @@
 
-            // Press Shift+F9 to start debugging your code. We have set one breakpoint
-            // for you, but you can always add more by pressing Ctrl+F8.
-            System.out.println("i = " + i);
-        }
+Выведите в консоль значение каждой переменной в формате «Значение переменной … с типом … равно …».*/
+    int valueInt=123;
+    byte valueByte=5;
+    short valueShort=1;
+    long valueLong=-666L;
+    float valueFloat=3.1F;
+    double valueDouble=3.14;
+        System.out.println("Значение переменной valueInt, с типом int, = "+valueInt+
+            "\nЗначение переменной valueByte, с типом byte, = "+valueByte+
+            "\nЗначение переменной valueSort, с типом short, = "+valueShort+
+            "\nЗначение переменной valueLong, с типом long, = "+valueLong+
+            "\nЗначение переменной valueFloat, с типом float = "+valueFloat+
+            "\nЗначение переменной valueDooble, с типом dooble, = "+valueDouble);
+        System.out.println("==================================================");
+    /*Задача 2
+Ниже дан список различных значений. Инициализируйте переменные, используйте изученные ранее типы переменных.
+Значения:
+27.12
+987 678 965 549
+2,786
+569
+-159
+27897
+67  */
+    float valueFloat2=27.12F;
+    long  valueLong2=987678965549L;
+    double valueDouble2=2.786;
+    short valueShort2=569;
+    int valueInt2=-159;
+    long valuesLong2=27897L;
+    byte valueByte2=67;
+    /*в консоль ,вроде, не надо выводить.*/
+        System.out.println("=====================================================");
+        /*Три школьных учителя, Людмила Павловна, Анна Сергеевна и Екатерина Андреевна, ведут три класса.
+        У Людмилы Павловны — 23 ученика , у Анны Сергеевны — 27 учеников, у Екатерины Андреевны — 30 учеников.
+        Три учительницы закупили все вместе 480 листов бумаги на все три класса. Посчитайте, сколько
+        достанется листов каждому ученику.
+        Результат задачи выведите в консоль в формате: «На каждого ученика рассчитано … листов бумаги».*/
+
+    int studentsClassA=23;
+    int studentsClassB=27;
+    int studentsClassG=30;
+    int countSheetsPaper=480;
+    int countStudents=studentsClassA+studentsClassB+studentsClassG;
+    int countSheetsForStudent=countSheetsPaper/countStudents;
+        System.out.println("На каждого ученика рассчитано "+countSheetsForStudent+" листов бумаги");
+        System.out.println("=====================================================");
+    /*Задача 4
+Производительность машины для изготовления бутылок — 16 бутылок за 2 минуты.
+Какая производительность машины будет:
+
+за 20 минут,
+в сутки,
+за 3 дня,
+за 1 месяц?
+Рассчитывайте производительность работы машины в том случае,
+если она работает без перерыва заданный промежуток времени.
+
+Результаты подсчетов выведите в консоль в формате: «За … машина произвела … штук бутылок».*/
+    byte countBottles=16;
+    int bottleMinetes=countBottles/2;
+    int bottleDay=bottleMinetes*1440;
```

Comment:
откуда 1440? никогда не нужно считать ничего в уме, для этого как раз пишутся программы:
```suggestion
    int bottleDay = 24 * 60 * bottleMinetes;
```